workout.py

```diff
@@ -116,7 +116,8 @@ def run_workout(workout, debug=False):
     gw = WorkoutGenerator(ses)
     wo = gw.generate_workout(config)
 
-    run_workout(wo, args.debug)
+    # TODO: Find better way of storing the player than sending it in all the method calls
```

Comment:
You could make a global variable since it's just used in this module

Reply:
yeah, I think you're right. That feels cleaner. Fixed